imagetest/testworkflow.go

```diff
@@ -272,8 +272,19 @@ func finalizeWorkflows(ctx context.Context, tests []*TestWorkflow, zone, gcsPref
 
 		twf.wf.Zone = zone
 
-		twf.wf.Sources["wrapper"] = testWrapperPath
-		twf.wf.Sources["testpackage"] = fmt.Sprintf("/%s.test", twf.Name)
+		arch := "amd64"
+		if machineType != "" {
+			createVMsStep := twf.wf.Steps[createVMsStepName]
+			for _, vm := range createVMsStep.CreateInstances.Instances {
+				vm.MachineType = machineType
+			}
+			if strings.HasPrefix(machineType, "t2a") {
+				arch = "arm64"
+			}
```

Comment:
Is this logic going to hold forever? I'm not very familiar with how machine types map to the architecture.

Reply:
t2a will always be ARM, but there may be new ARM types in the future. the Right Thing To Do™ here is to use compute API and read arch from machine type, but that's not yet available in prod API, and would also be the first direct use of compute API in this code.

Reply:
SGTM. For tracking, I would personally create a bug to update this code once the API is available.